mix_common.exs

```diff
@@ -157,13 +158,10 @@ defmodule Antikythera.MixConfig do
   end
 
   defp default_configs() do
-    [
-      # Limit port range to be used for ErlangVM-to-ErlangVM communications.
-      kernel: [
-        inet_dist_listen_min: 6000,
-        inet_dist_listen_max: 7999,
-      ],
+    # Loading log translator
+    Code.eval_file("erlang_log_translator.ex", Path.join(__DIR__, "core/util"))
```

Comment:
There is no `:translators` in [the config doc](https://hexdocs.pm/logger/1.12.3/Logger.html#module-configuration), and [the Translator module doc](https://hexdocs.pm/logger/1.12.3/Logger.Translator.html#content) mentions only `Logger.add_translator/1`. Therefore, I guess `:translators` isn't an API. This is the reason why warning is printed.

Since `AntikytheraCore.ErlangLogTranslator` should be used at runtime instead of compile time, I think it would be better to add `AntikytheraCore.ErlangLogTranslator` at `AntikytheraCore.start/2`.

Reply:
Thank you for your suggestion.
Fixed at 263be31 and 3a2ea42.
This change prevents to skip some of the logs before `AntikytheraCore` starts.

Reply:
> This change prevents to skip some of the logs before `AntikytheraCore` starts.

I found that many (200~300 lines) noisy logs show in terminal during gear's `mix test` and `iex -S mix`.
I think gear developers will have difficulty navigating the terminal.
Could you fix this problem?
(Sorry, I currently don't have any suggestions.)

Reply:
04463a7 may be the simplest solution.

Reply:
I think it's better to avoid setting `:translators` if not needed (because it's undocumented).
How about defining the version-dependent part separately and appending it to `:logger` configs?

```elixir
# TODO: Remove after requiring Elixir 1.11+
logger_configs_for_compatibility =
  if Version.match?(System.version(), "~> 1.11") do
    [
      # To suppress progress reports during start-up in development environments
      level: (if Mix.env() == :prod, do: :info, else: :notice)
    ]
  else
    [
      level: :info,
      translators: [{AntikytheraCore.ErlangLogTranslator, :translate}]
    ]
  end
```

Reply:
It looks good.
Fixed at d8a5606.